src/main/webapp/index_script.js

```diff
@@ -72,15 +74,16 @@ class SearchArea {
     this.searchArea.appendChild(this.organizationPopupArea);
   }
 
-   /* 
-     * This async function gets a default list of organization names to display when the dom content
-     * loads by not passing any parameters to /list-organizations
-     */
+  /* 
+    * This async function gets a default list of organization names to display when the dom content
+    * loads by not passing any parameters to /list-organizations
+    */
   async requestAndDisplayOrganizations() {
+    //TODO: fetch organizations with param
     const response = await fetch(`/list-organizations`);
-    const organizationNames = await response.json();
-    for (let i = 0; i < organizationNames.length; i++) {
-      this.organizationList.appendChild(this.createOrganization(organizationNames[i]));
+    const organizations = await response.json();
+    for (let i = 0; i < organizations.length; i++) {
```

Comment:
for (const org of organizations) {
   this.organziationList.appendChild(org);
}